test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -69,6 +69,18 @@ public EndToEndTestsBase(TTestFixture fixture)
             Assert.Equal(doc.Id, id);
         }
 
+        protected async Task NotificationHubTest()
+        {
```

Comment:
For now, just verifying the call succeeds. Will add more verification later. There might be a way to call NH api to check if sendAsync was called.